produce/lib/produce/options.rb

```diff
@@ -137,7 +137,19 @@ def self.available_options
                                      default_value: CredentialsManager::AppfileConfig.try_fetch_value(:itc_team_name),
                                      verify_block: proc do |value|
                                        ENV["FASTLANE_ITC_TEAM_NAME"] = value.to_s
-                                     end)
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :merchant_name,
+                                      short_option: "-r",
+                                      env_name: "PRODUCE_MERCHANT_NAME",
+                                      description: "Name for the merchant that is created",
+                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :merchant_identifier,
```

Comment:
@KrauseFx good catch! I have fixed this up. I couldn't find a good existing example of passing the extra options when they aren't related to the regular command so can you please review my changes in `produce/commands_generator.rb`